samples/zigbee/light_switch/include/nus_cmd.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file nus_cmd.h
+ *
+ * @brief APIs for NUS Command Service.
+ */
+
+#ifndef __NUS_CMD_H__
+#define __NUS_CMD_H__
+
+/** @brief Type indicates function called when Bluetooth LE connection
+ *         is established.
+ *
+ * @param[in] item pointer to work item.
+ */
+typedef void (*nus_connection_cb_t)(struct k_work *item);
+
+/** @brief Type indicates function called when Bluetooth LE connection is ended.
+ *
+ * @param[in] item pointer to work item.
+ */
+typedef void (*nus_disconnection_cb_t)(struct k_work *item);
+
+/** @brief NUS command entry.
+ */
+typedef struct nus_entry_s {
```

Comment:
Zephyr (NCS) coding standard discourages using `typdef` when it can be avoided, consider sticking to `struct nus_entry` instead.

Reply:
fixed